### ISSUE TITLE
Missing package.json

### ISSUE THREAD

**Issue by Brainicism** [frankclong/Messenger-Analyzer#8]
`/frontend` is missing a package.json, for dependencies as well as the 'npm run dev' target.

**Comment by frankclong** [comment]
Good catch, I've added the package.json file and removed *.json from the .gitignore.
